setup.py

```diff
@@ -10,14 +10,14 @@
 setup(
     name='ctapipe_io_nectarcam',
     packages=find_packages(),
-    version='0.1',
+    version='0.1', #TODO implement versioning
```

Comment:
you might want to bump the version number before merging, however

Reply:
I was wondering about that, I left it on hold since @maxnoe made another PR to implement automated versioning. What is the the convention for versioning, this should be 0.2 I guess?

Reply:
Just leave it unchanged. It will be removed when we merge the other PR and from then on, we can use the git based versioning.